client/src/style.css

```diff
@@ -22,6 +22,8 @@ body {
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
   -webkit-text-size-adjust: 100%;
+  /* Kiwix iFrame will overlay 5em on the bottom */
+  margin-bottom: 7em;
```

Comment:
This is scrolled all the way to the bottom, but the bottom bar overlaps the content.
![image](https://github.com/openzim/freecodecamp/assets/2868/62238957-9cc6-4e04-b4ec-d7662a2c06b1)

It's not a huge issue, but I can't seem to easily fix it on my end. This is in Chrome on Linux (Version 114.0.5735.198 (Official Build) (64-bit))

Reply:
I saw the same issue with `kolibri`.
Looks like this is only a `kiwix-js` issue from my experience (no issue on kiwixserve typically).
@Jaifroid : are you aware of anything like this?

Reply:
OK then we can drop that hack. @benoit74 please open a kiwix-js ticket

Reply:
Sorry to be slow to reply -- didn't see the thread till now. We do have an issue on Kiwix JS with the bottom bar hiding content. No need to create a new one: https://github.com/kiwix/kiwix-js/issues/435 and https://github.com/kiwix/kiwix-js/issues/337.

I'd like to test a ZIM on Kiwix JS PWA (https://pwa.kiwix.org), which has some hackish workaround code with the bottom bar issue.

Reply:
@Jaifroid : from my tests with very few ZIMs on Kiwix JS PWA, I confirm the issue is "solved" on this platform, the bottom bar fades out as we slide down. I don't know what would happen if there is no slider because the page perfectly fits the viewport (extreme corner case, but, you know ...)